### PR TITLE
clean up the log at startup

### PR DESCRIPTION
**Pull request by gavinking**
Here's a few little things that make the boot logging look a lot more professional.

Please see the individual commits.

### CODE REVIEW

**Comment by gavinking** [comment]
OK, so this enabled a big simplification of the "services" package. That's much better!

File: example/build.gradle [hunk 1]
```diff
@@ -2,7 +2,12 @@ description = 'Hibernate Reactive Example'
 
 dependencies {
     implementation project(':hibernate-reactive-core')
+
+    // database drivers for PostgreSQL and MySQL
     runtimeOnly 'io.vertx:vertx-pg-client:3.9.1'
     runtimeOnly 'io.vertx:vertx-mysql-client:3.9.1'
+
+    // logging
+    runtimeOnly 'org.slf4j:slf4j-log4j12:1.7.30'
```
Comment (Sanne) [inline example/build.gradle:11]: just curious about the choice of slf4j + log4j ... why slf4j? Is vertx using it?

File: hibernate-reactive-core/src/main/java/org/hibernate/reactive/boot/impl/ReactiveServiceRegistryInitializer.java [hunk 2]
```diff
@@ -20,9 +20,9 @@
 	@Override
 	public void contribute(StandardServiceRegistryBuilder serviceRegistryBuilder) {
 		serviceRegistryBuilder.addInitiator( VertxInstanceInitiator.INSTANCE );
-		serviceRegistryBuilder.addInitiator( DummyConnectionProviderInitiator.INSTANCE );
-		serviceRegistryBuilder.addInitiator( ReactiveJdbcEnvironmentInitiator.INSTANCE );
-		serviceRegistryBuilder.addInitiator( ReactiveTransactionCoordinatorBuilderInitiator.INSTANCE );
+		serviceRegistryBuilder.addInitiator( NoJtaPlatformInitiator.INSTANCE );
```
Comment (Sanne) [inline hibernate-reactive-core/src/main/java/org/hibernate/reactive/boot/impl/ReactiveServiceRegistryInitializer.java:23]: nice!

File: hibernate-reactive-core/src/main/java/org/hibernate/reactive/boot/service/NoJdbcConnectionProviderInitiator.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+public class NoJdbcConnectionProviderInitiator implements StandardServiceInitiator<ConnectionProvider> {
+
+	public static final NoJdbcConnectionProviderInitiator INSTANCE = new NoJdbcConnectionProviderInitiator();
+
+	@Override
+	public ConnectionProvider initiateService(Map configurationValues, ServiceRegistryImplementor registry) {
+		ConnectionProvider provider = ConnectionProviderInitiator.INSTANCE.initiateService(configurationValues, registry);
+		if (provider instanceof DriverManagerConnectionProviderImpl) {
+			registry.getService( SchemaManagementTool.class )
+					.setCustomDatabaseGenerationTarget( new ReactiveGenerationTarget(registry) );
```
[the remark this answers is not in the document]
Comment (gavinking) [reply hibernate-reactive-core/src/main/java/org/hibernate/reactive/boot/service/NoJdbcConnectionProviderInitiator.java:28]: OK. If you have another suggestion of how to do this I can change it.

(But it's working fine for now.)

File: example/build.gradle (hunk 1 above)
Comment (gavinking) [reply example/build.gradle:11]: Um ... dunno, it's what reactive core was set up to use. Wasn't my choice...

**Comment by Sanne** [comment]
It's great - thanks!  Just push it as soon as CI is happy.

One question though: I couldn't work on HR today but I'll try to work this weekend - I will probably need need to refactor 100% of the services: could you refrain from working in the same area for a little?

File: example/build.gradle (hunk 1 above)
Comment (Sanne) [reply example/build.gradle:11]: ok, np I was just curious.

**Comment by gavinking** [comment]
> could you refrain from working in the same area for a little?

OK.

**Comment by gavinking** [comment]
WTF is up with the CI?

**Comment by Sanne** [comment]
> WTF is up with the CI?

Looks like some github service is down - Let's give it a bit of time then we can try running it again.

**Comment by aguibert** [comment]
@gavinking github is having issues. I got an error 500 just a minute ago trying to leave a comment on this PR.
See: https://www.githubstatus.com/

Probably just wait a few mins and try again

**Pull request merged**